src/main/java/Duke.java

```diff
@@ -1,10 +1,148 @@
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import java.util.Scanner;
+import java.util.Arrays;
+import java.util.List;
+import java.util.ArrayList;
+
+
 public class Duke {
+    private static Path filePath = Paths.get("data", "duke.txt");
+    private static Store store = new Store(filePath);
+
+    private static ArrayList<Task> tasks;
+    private static int numTasks = 0;
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        runDuke();
+    }
+
+    public static void greeting() {
+        System.out.println(delimiter());
+        System.out.println("Hello! I'm Duchess");
+        System.out.println("What can I do for you?");
+        System.out.println(delimiter() + "\n");
+    }
+
+    public static String delimiter() {
+        return "___________________________________________________";
+    }
+
+    public static void runDuke() {
+        tasks = store.getStoredTasks();
+        numTasks = tasks.size();
+        greeting();
+
+        Scanner sc = new Scanner(System.in);
+
+        while (true) {
+            String input = sc.nextLine();
+
+            try {
+                String command = getCommand(input);
+                if (command.equals("bye")) {
```

Comment:
command 'bye' should be treated equally as other command to avoid confusion. For instance, can throw exception when doInstruction("bye").